### PR TITLE
Resolve conflicts of PR for https://github.com/wso2/carbon-apimgt/pull/6358 

### PR DESCRIPTION
**Pull request by ruwiniwj**
## Purpose
Resolve conflicts and create new PR for https://github.com/wso2/carbon-apimgt/pull/6358

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/mappings/ThrottlingPolicyMappingUtil.java [hunk 1]
```diff
@@ -36,12 +36,12 @@
     /**
      * Converts a List object of Tiers into a DTO
      *
-     * @param throttlingPolicyList a list of ThrottlingPolicy objects
+     * @param throttlingPolicyList a list of ThrottlingPolicy_ objects
```
Comment (isharac) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/mappings/ThrottlingPolicyMappingUtil.java]: Isn't this throttlingPolicyList a list of Tier objects

Comment (isharac) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/mappings/ThrottlingPolicyMappingUtil.java]: doc comment for policyLevel is missing

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/mappings/ThrottlingPolicyMappingUtil.java [hunk 2]
```diff
@@ -101,11 +101,11 @@ public static void setPaginationParams(ThrottlingPolicyListDTO tierListDTO, Stri
     /**
      * Converts a Tier object into TierDTO
      *
-     * @param throttlingPolicy ThrottlingPolicy object
+     * @param throttlingPolicy ThrottlingPolicy_ object
```
Comment (isharac) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/mappings/ThrottlingPolicyMappingUtil.java]: Tier object?

Comment (ruwiniwj) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/mappings/ThrottlingPolicyMappingUtil.java]: fixed with https://github.com/wso2/carbon-apimgt/pull/6797

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/mappings/ThrottlingPolicyMappingUtil.java (hunk 1 above)
Comment (ruwiniwj) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/mappings/ThrottlingPolicyMappingUtil.java]: https://github.com/wso2/carbon-apimgt/pull/6797

**Pull request merged**